ansible/roles/test/files/acstests/everflow_policer_test.py

```diff
@@ -152,14 +172,16 @@ def match_payload(pkt):
         self.dataplane.flush()
 
         count = 0
+        testutils.send_packet(self, self.src_port, self.base_pkt, count=self.NUM_OF_TOTAL_PACKETS)
         for i in range(0,self.NUM_OF_TOTAL_PACKETS):
-            testutils.send_packet(self, self.src_port, self.base_pkt)
             (rcv_device, rcv_port, rcv_pkt, pkt_time) = testutils.dp_poll(self, timeout=0.1, exp_pkt=masked_exp_pkt)
             if rcv_pkt is not None and match_payload(rcv_pkt):
                 count += 1
             elif count == 0:
                 print "The first mirrored packet is not recieved"
```

Comment:
One small note, rather then comment.
Previous implementation had a strict check that first mirrored packet is not rate limited by the policer.
If this is OK I suggest to remove "elif count == 0" block

Reply:
@stepanblyschak The reason why i didn't touch it, just to have an additional informative message. But you are right, we can completely remove it.